run-headless-chromium.js

```diff
@@ -118,10 +118,11 @@ var userDataDir = path.join(os.tmpdir(), 'chromium_headless_user_data_directory'
     }
 })();
 
-
 // Start the virtual X Framebuffer
 var xvfb = new Xvfb({
     silent: true,
+    // Check environment variable for reuse
```

Comment:
Remove this comment, it doesn't add anything that is not already obvious from looking at the code.